src/subsplit_dag.hpp

```diff
@@ -487,21 +489,17 @@ class SubsplitDAG {
   // Expand dag_edges_ and parent_to_child_range_ with leaf subsplits at the end.
   void AddLeafSubsplitsToDAGEdgesAndParentToRange();
 
+  void StoreEdgeIds();
```

Comment:
Is this really still here?

Reply:
I was thinking about GetStorage(), which I moved to an anonymous namespace. I'll fix this too